front/app/modules/commercial/analytics/admin/hooks/useProposals/typings.ts

```diff
@@ -0,0 +1,12 @@
+import { XlsxData } from 'components/admin/ReportExportMenu';
+import { Stat } from '../../typings';
+
+export interface ProposalsChartData {
```

Comment:
```suggestion
export interface ProposalsStats {
```

(sorry super nitpicky but it's not really a chart)